### ISSUE TITLE
Failed to Launch DAPP

### ISSUE THREAD

**Issue by anupam-git** [LiskHQ/lisk#221]
Hi,

I am a freelancer currently working for a client. I am trying to integrate Lisk into a project.

So far, I have installed the lisk server correctly and created a dapp using "lisk-cli dapps -a".

However after restarting the lisk server(after creating the dapp), whenever I try to open http://localhost:7000/dapps/15084714724197785430 (15084714724197785430 is my dapp id) I am getting redirected to the login page.

Then to see the error, I manually started the node server by "node app.js". On examining the log generated, I ca see that the dapp was not launched. Not other information is there for me to solve this issue. This was the error message "Failed to launch application 15084714724197785430: Dapp not found".

Please help me to solve me this issue.

Thanks,
Anupam Basak.
Email : anupam.basak27@gmail.com

**Comment by karmacoma** [comment]
Please check the dapp id is registered on your local testnet, see `genesisBlock.json` is the same as configured in `autoexec` within `config.json`.

**Comment by ByronAP** [comment]
are we assuming that you registered the dapp in the node?

**Comment by karstenjakobsen** [comment]
I'm having the same error. Have followed the tutorial and end up in the same sutuation. I have tried with both a Debin 8 and Ubuntu 14 og 16.

This is my app.log:

lisk@lisk:~/lisk-test$ bash lisk.sh restart
√ Lisk stopped successfully.
√ Postgresql stopped successfully.
√ Postgresql started successfully.
√ Lisk started successfully.
lisk@lisk:~/lisk-test$ tail -f app.log
NOTICE:  relation "mem_accounts2u_delegates" already exists, skipping
NOTICE:  relation "mem_accounts2multisignatures" already exists, skipping
NOTICE:  relation "mem_accounts2u_multisignatures" already exists, skipping
info 2016-07-05 06:37:39 Modules ready and launched
info 2016-07-05 06:37:39 Blocks 137582
info 2016-07-05 06:37:39 Blockchain ready
error 2016-07-05 06:37:39 Failed to load delegates: Account 744147084f03b85c585541bad8f29aa9d2fac0ede514c52df30ee7b53f485246 not found
info 2016-07-05 06:37:39 Peers ready, stored 128
error 2016-07-05 06:37:40 Application not found
Failed to launch application 13035085409444566396: Dapp not found

**Comment by anupam-git** [comment]
Hi.

I solved this issue.

When I manually ran the server by "node app.js" I saw that some insert statements failed to run and hence the transaction failed.

So after installing the Dapp I dropped the database and created a database with the same name without any table.

"dropdb lisk_main"
"createdb lisk_main"

Now when I restarted the server, the problem was gone. The dapp was opening. There was no error shown in the terminal.

I don't know what actually happened for which data was not getting inserted into the table. It seems that it is an issue with LISK.

Thanks.
